### PR TITLE
Add editorial to problem 1289 (en): Issue 196

### PR DESCRIPTION
**Pull request by RobinHood3082**
issue: 196

### CODE REVIEW

File: 1289/en.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+## Solution
+
+### Naive Approach
+Let's calculate for each prime (until *n*) the maximum power *a* of this prime *p*, so that *p<sup>a</sup> <= n*. For example, if *n = 100*, prime *5* has highest power of *log<sub>5</sub>100*, i.e. the maximum number of time we can divide *100* by *5*, which is *2*. You can see, *5<sup>2</sup> = 25* which is less than *100*, and *5<sup>3</sup> = 125* which is greater than *100*. So here, for *n = 100*, *p = 5* we get *a = 2*.
+
+So, for each prime *p<sub>i</sub>* until *n*, we calculate *a<sub>i</sub>* , which is the maximum power of that prme till *n*. Then, *lcm =  p<sub>1</sub><sup>a<sub>1</sub></sup> . p<sub>2</sub><sup>a<sub>2</sub></sup> . ... . p<sub>k</sub><sup>a<sub>k</sub></sup>* , which is the required answer.
+
+#### Problem with Naive Solution
+*n* can be equal to *10<sup>8*, and *T* can be equal to *10<sup>4</sup>* . There are rougly *k = 6x10<sup>6</sup>* primes till  *10<sup>8* . So for each case, if we intend to use the naive solution, the complexity for each case would be *O(k log n)*, where *k* is the number of prime factors till *n*. So, total complexity will be *O(Tk log n)* , which won't pass the time limit of  4s.
```
Comment (showrav-ansary) [inline 1289/en.md:23]: There is a minor typo. Please `rougly` to `roughly`.

**Comment by showrav-ansary** [comment]
@RobinHood3082 Dear contributor. You have done an excellent step to step clarification in a very solid manner! Excellent 💯  Please fix the minor typo. Will merger as soon as you correct it! Thank you in advance ❤️ Keep contributing more ❤️

**Comment by showrav-ansary** [comment]
@RobinHood3082 Dear contributor. You have done an excellent step by step clarification in a very solid manner! Excellent 💯  Please fix the minor typo. Will be merged as soon as you correct it! Thank you in advance ❤️ Keep contributing more ❤️

File: 1289/en.md (hunk 1 above)
Comment (showrav-ansary) [inline 1289/en.md:23]: There is a minor typo. Please change `rougly` to `roughly`.

Comment (RobinHood3082) [reply 1289/en.md]: I fixed the typo, thanks for letting me know!

**Comment by showrav-ansary** [comment]
@RobinHood3082 Dear contributor, thank you very much 🥇

**Pull request merged**